cypress/integration/AutoTest.js

```diff
@@ -18,5 +25,26 @@ describe('AutoTest Project', () => {
         // After search entered into search bar there should be on a new URL which includes '/commercial-ice-machines'
         cy.url().should('include', '/commercial-ice-machines')
 
+
+          // Tests vendor filter function for "Manitowoc" and verifies product count in vendor filter matches count in product list
+        
+
+        // Searches webpage for "Full-Dice Ice Machines" link. Clicks link and checks new webpage for proper address
+        cy.get('.item-categories').contains('Full-Dice Ice Machines').click()
+        cy.url().should('include', '/full-dice')
+
+        // Clicks the "Manitowoc" vendor checkbox using the Manitowoc brand value 
+        cy.get('.js-filter-value').check('09e51896-5085-42fc-9265-c23b5520db07')
+
+        // Waits for new webpage to load
+        cy.wait(1000)
```

Comment:
1000 may not be enough if CI server has slow connection for example. Better way - do not use waits with specific times, but wait some changes on page or make sure some elements disappered or became visible. For example: when vendor filter is applied the button 'Clear filters' becomes visible or page header changes from Full Size Ice Cube Machines to Manitowoc Full Size Ice Cube Machines.